src/visitor.ts

```diff
@@ -20,10 +20,9 @@ export type VaryOptions = {
 };
 
 export type AbOptions = {
-  callback: (assignment: boolean | string) => void;
+  callback: (assignment: boolean) => void;
   context: string;
-  trueVariant: boolean | string;
```

Comment:
This is used here: https://github.com/Betterment/test_track_js_client/blob/087b112c71ce942a1b39b5ad9ee3f3670516520f/src/abConfiguration.ts#L45

`boolean | string` makes it so that this is _not_ optional, which contradicts the docs (!)